src/main/scala/org/zalando/spearheads/innkeeper/Rejections.scala

```diff
@@ -0,0 +1,87 @@
+package org.zalando.spearheads.innkeeper
+
+import akka.http.scaladsl.model.{StatusCodes, StatusCode}
+import akka.http.scaladsl.server.Rejection
+
+/**
+ * @author dpersa
+ */
+sealed trait InnkeeperRejection {
+
+  /**
+   * @return the request method followed by the request path
+   */
+  def requestDescription: String
+  def statusCode: StatusCode
+  def message: String
+  def code: String
+}
+
+object Rejections {
+
+  case class RouteNotFoundRejection(requestDescription: String) extends Rejection with InnkeeperRejection {
+    def statusCode: StatusCode = StatusCodes.NotFound
+    def message: String = "Route not found"
+    def code: String = "RNF"
+  }
+
+  case class IncorrectTeamRejection(requestDescription: String) extends Rejection with InnkeeperRejection {
+    def statusCode: StatusCode = StatusCodes.Forbidden
+    def message: String = "User not member of a team which is not allowed to do this call"
+    def code: String = "ITE"
+  }
+
+  case class TeamNotFoundRejection(requestDescription: String) extends Rejection with InnkeeperRejection {
+    def statusCode: StatusCode = StatusCodes.Forbidden
+    def message: String = "Team Not Found. You need to be part of a tech team to do this call"
+    def code: String = "TNF"
+  }
+
+  case class NoUidRejection(requestDescription: String) extends Rejection with InnkeeperRejection {
+    def statusCode: StatusCode = StatusCodes.Forbidden
+    def message: String = "The token you provided doesn't have an associated UID. Is this a service token?"
+    def code: String = "NUID"
+  }
+
+  case class InvalidRouteNameRejection(requestDescription: String) extends Rejection with InnkeeperRejection {
+    def statusCode: StatusCode = StatusCodes.BadRequest
+    def message: String = """"Invalid route name. Route names follow the pattern: "[a-zA-Z][a-zA-Z0-9_]*""""
```

Comment:
What do four-quote strings do in scala? Or do you actually want the inner most quotes to be part of the massage?